packages/extension/src/components/pages/AddNewTrustline/AddNewTrustline.tsx

```diff
@@ -278,14 +352,22 @@ export const AddNewTrustline: FC = () => {
           </>
         }
         transaction={TransactionStatus.Rejected}
+        {...(params.inAppCall ? { onClick: () => navigate(HOME_PATH) } : {})}
       />
     );
   }
 
   if (errorValue) {
```

Comment:
```suggestion
  if (errorValue && !params.inAppCall) {
```

Reply:
We still want to return <AsyncTransaction ... if isValidIssuer = false, even if it's not an inApp call